services/src/api/handlers/workflows.rs

```diff
@@ -621,6 +622,7 @@ async fn vector_stream_websocket<C: ApplicationContext>(
         spatial_bounds: query.spatial_bounds,
         time_interval: query.time_interval.into(),
         spatial_resolution: query.spatial_resolution,
+        selection: Default::default(), // TODO
```

Comment:
do what?

Reply:
I made it more conrete e6b9479658da849934df78cc2b2b328aad6511b9